package.json

```diff
@@ -5,6 +5,14 @@
   "author": "{{{ author }}}",
   "license": "ISC",
   "repository": "https://github.com/{{ owner }}/{{ repo }}.git",
+  "homepage": "https://probot.github.io/apps/{{ name }}/",
+  "bugs": "https://github.com/probot/{{ name }}/issues",
```

Comment:
homepage assumes that the App will get listed on the website which is not necessarily true, so maybe assume they start out with their readme as homepage: 
`"homepage": "https://github.com/{{ owner }}/{{ repo }}/blob/master/README.md"`
but that's also kinda gross, so maybe just the same as the `repository` field's link?

bugs also assume that the repo of the app will be within the probot org which is actually pretty unlikely, the probot org is only home to a few select "OG" probot apps. so maybe:
 `"bugs": "https://github.com/{{ owner }}/{{ repo }}/issues",`